src/apiCalls.js

```diff
@@ -13,4 +13,19 @@ export function getData () {
   })
 }
 
-export default {getData}
\ No newline at end of file
+export function getSingleMovie (movieId) {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieId}`)
+  .then((response) => {
+    if (!response.ok) {
+      throw new Error(`${response.status}`)
+    } else {
+      return response.json()
+    }
+  })
+  .catch((err) => {
```

Comment:
Nice job using throw error - There's no need to have the catch block here. You can move the catch block to where you're calling this function within the componentDidMount() method.